webhook_features/src/features/summary_comment.rs

```diff
@@ -286,23 +283,20 @@ impl AckType {
     }
 }
 
-macro_rules! multi_vec {
-    ($([$($key:literal),+] => $value:expr);*) => {
-        vec![
-            $($(($key, $value)),*),*
-        ]
-    };
-}
-
 lazy_static! {
-    static ref ACK_PATTERNS: Vec<(&'static str, AckType)> = multi_vec![
-        ["code review ack", "cr ack", "cr-ack", "crack"] => AckType::Ack;
-        ["ack", "utack", "tack"] => AckType::Ack;
-        ["concept ack", "concept-ack", "conceptack", "ack"] => AckType::ConceptAck;
-        ["concept nack", "concept-nack", "conceptnack"] => AckType::ConceptNack;
-        ["approach ack", "approach-ack", "approachack"] => AckType::ApproachAck;
-        ["approach nack", "approach-nack", "approachnack"] => AckType::ApproachNack;
-        ["nack"] => AckType::ConceptNack
+    static ref ACK_PATTERNS: Vec<(&'static str, AckType)> = vec![
+        // approach ACK, approach-ack, approachack
+        (r".*\b(Approach ACK)\b.*", AckType::ApproachAck),
+        // approach NACK, approach-nack, approachnack
+        (r".*\b(Approach NACK)\b.*", AckType::ApproachNack),
+        // NACK
+        (r".*\b(NACK)\b", AckType::ConceptNack),
+        // concept ACK, concept-ack, conceptack, ack
+        (r".*\b(Concept ACK)\b.*", AckType::ConceptAck),
+        // ACK, utACK, tACK [commit hash]
+        (r".*\b(?:re)?(ACK|utACK|tACK)((\s)*[0-9a-f]{6,40})\b.*", AckType::Ack),
```

Comment:
```suggestion
        (r".*\b(re)?(ACK|utACK|tACK)(\s*)([0-9a-f]{6,40})\b.*", AckType::Ack),
```

maybe move the whitespace out of the group and then also remove `trim()` below?

Reply:
This changes the capture groups order and messes with the commit extraction :grimacing:

Reply:
Yes, I presume you'll need to change `2` to `3` or so?

Reply:
Doesn't seem to fix it. Unfortunately, I won't have time to work on this PR further this weekend so I suggest to fix the nits later?